src/Xamarin.Android.Build.Tasks/Xamarin.Android.Application.targets

```diff
@@ -35,4 +35,30 @@ Copyright (C) 2019 Microsoft Corporation. All rights reserved.
     </GetAndroidPackageName>
     <Exec Command="&quot;$(AdbToolPath)adb&quot; $(AdbTarget) shell am force-stop &quot;$(_AndroidPackage)&quot;" />
   </Target>
+  <PropertyGroup>
+    <AndroidAotCustomProfilePath Condition=" '$(AndroidAotCustomProfilePath)' == '' ">custom.aprof</AndroidAotCustomProfilePath>
+    <AndroidAotProfilerPort Condition=" '$(AndroidAotProfilerPort)' == '' ">9999</AndroidAotProfilerPort>
+    <_BeginAotProfilingDependsOnTargets>
+      _SetupAotProfiling;
+      Build;
```

Comment:
@jonathanpeppers should this be conditional based on if we are in the IDE or not like we do for the install target?

Reply:
I think what is here seems OK, because it is a specific target not in the devloop.

If it's the condition here: https://github.com/xamarin/xamarin-android/blob/fd64aaff618f5f818831bdef844876e138fec7a0/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L3097-L3110

This was just a performance thing, we shouldn't need to worry about this custom target, then?